client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
-          You can now evaluate a DMN decision in a Business Rules Task.
-          Specify the called decision (DMN) by its ID and assign the result to a variable via the properties panel.
-          Learn more about this feature in the <a href="https://docs.camunda.io/docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks/">Camunda Cloud docs on Business Rule Tasks.</a>
+          <h4>Support for Camunda 8</h4>
+          With this release, we added support for Camunda 8.<br />
+          You can now model BPMN, and DMN diagrams as well as Forms for Camunda 8.
+          Using plugins, you can now also customize the Modeler specifically for Camunda 8.
+          We will now also use a more concise naming: <i>Camunda Platform</i> becomes Camunda 7
+          and <i>Camunda Cloud</i> becomes Camunda 8.<br />
+          Learn more about Camunda 8 in general <a href="toBeAdded">in the Camunda 8 blogpost.</a>
         </li>
         <li>
           <h4>Improved overall UI to become a "Visual IDE"</h4>
-          New welcome screen allows to easily create Platform and Cloud diagrams and forms.
-          The tab bar displays icons according to the file type and provides a new file menu via the plus button.
-          The properties panel has been completely redesigned. Tabs have been replaced with sections which can be opened and closed individually.
-          A granular breakdown with clear titles allows to identify relevant topics at a glance.
+          We re-worked the entire Modeler, including welcome screen, properties panel, status bar, tab bar, and bottom panel
+          following our visions of a "Visual IDE".<br />
+          Learn more about the changes <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released/">in this blog post</a>.
+        </li>
+        <li>
+          <h4>Process Debugging for Camunda 8 BPMN diagrams</h4>
+          The Modeler will now support you when automating BPMN processes with Camunda 8 by showing implementation errors,
+          and giving hints how to fix them.
         </li>
         <li>
           <h4>Support drilldown into collapsed subprocesses</h4>
-          With this release, we improved modeling of collapsed subprocesses for Camunda Platform.<br />
-          You can model collapsed subprocesses through the command pallet or by collapsing an expanded subprocess.
-          Use the blue drilldown icon to view and model the contents of the subprocess.
+          In Camunda 7 BPMN diagrams, you can now model collapsed subprocesses by
+          collapsing an expanded subprocess. Use the blue drilldown icon to view and model the contents of the subprocess.
         </li>
         <li>
-          <h4>Process Debugging for BPMN</h4>
-          The Modeler will now validate and show if BPMN elements are not supported in the target platform.
+          <h4>Camunda 8 support and new features for element templates</h4>
```

Comment:
As Andi already stated, "Camunda 8 support" is repeated here.

We could reorder the sections (move this below "Camunda 8 support") and rename it to "Powerful new features for element templates". But I would favor something more specific here

Reply:
Corrected. Ty